.github/workflows/deploy-examples.yml

```diff
@@ -0,0 +1,68 @@
+name: Deploy example apps to Fly
+
+on:
+  # Allow calling this workflow from other workflows
+  workflow_call:
+    inputs:
+      version:
+        type: string
+        description: "Wasp version to use for deployment (default to latest)"
+        required: false
+    secrets:
+      FLY_API_TOKEN:
+        description: "Fly API token for deploying apps"
+        required: true
+
+  # Allow calling this workflow manually from the GitHub UI
+  workflow_dispatch:
+    inputs:
+      version:
+        description: "Wasp version to use for deployment (default to latest)"
+        required: false
+
+  # TODO: REMOVE THIS BEFORE MERGING, JUST FOR TESTING
+  push:
+    branches:
+      - 2831-automatically-deploy-all-example-apps-on-release
+
+concurrency:
+  group: "deploy-examples"
+  cancel-in-progress: true
+
+jobs:
+  deploy:
+    strategy:
+      matrix:
+        projects:
+          - name: waspleau
+            dir: examples/waspleau
+            url: https://waspleau-app-client.fly.dev/
+
+          - name: websockets-realtime-voting
+            dir: examples/websockets-realtime-voting
+            url: https://websockets-voting-client.fly.dev/
+
+    name: Deploy ${{ matrix.projects.name }}
+
+    runs-on: ubuntu-latest
+
+    environment:
+      name: ${{ matrix.projects.name }}-production
+      url: ${{ matrix.projects.url }}
```

Comment:
This allows us to have a history of deployments at https://github.com/wasp-lang/wasp/deployments/waspleau-production and https://github.com/wasp-lang/wasp/deployments/websockets-realtime-voting-production

Reply:
Nice!